src/js/view/controls/shortcuts-tooltip.ts

```diff
@@ -97,16 +115,17 @@ export default function (container, api, model, onVisibility) {
         shortcutToggleUi.destroy();
     };
 
-    const documentClickHandler = (e) => {
-        if (!/jw-shortcuts|jw-switch/.test(e.target.className)) {
+    const documentClickHandler = (e: Event) => {
+        const target = e.target as HTMLElement;
+        if (!/jw-shortcuts|jw-switch/.test(target.className)) {
             close();
         }
     };
 
-    const toggleClickHandler = (e) => {
-        const toggle = e.currentTarget;
+    const toggleClickHandler = (e: Event) => {
+        const toggle = e.currentTarget as HTMLElement;
         const isChecked = toggle.getAttribute('aria-checked') === 'true' ? false : true;
```

Comment:
Can we get rid of this ternary op?
```suggestion
        const isChecked = toggle.getAttribute('aria-checked') !== 'true';
```